test.html

```diff
@@ -164,12 +281,72 @@ <h2>Test drive</h2>
 		});
 }
 
+/**
+ * Loads a plugin including it css file
+ *
+ * @param {string} pluginName the plugin to load
+ * @type {Promise} the promise which resolves as soon as plugin is loaded
+ */
+function loadPlugin (pluginName)
+{
+	var plugin = components.plugins[pluginName];
+	if (!plugin){
+		return;
+	}
+
+	var pluginPath = components.plugins.meta.path;
+
+	return new Promise(function (resolve) {
+		pluginPath = pluginPath.replace(/{id}/g, pluginName);
+
+		$u.script(pluginPath + '.js', function() {
+			this.setAttribute('data-type', 'plugin');
+
+			var hasCss = !!!plugin.noCSS;
```

Comment:
You can just use `!` here.

Reply:
What about if an object doesn't have `noCSS`? !undefined will be true - which is wrong

Reply:
But `!!!undefined` is also `true`

Reply:
Omg, indeed, sorry about that. So it will be `!!plugin.noCSS`. So for undefined (when noCSS attribute isn't provided) it will be false, while for true - it will be true